File: connectors/connector-trello/src/test/scala/com/taskadapter/connector/trello/TrelloTestConfig.scala
package com.taskadapter.connector.trello

import com.taskadapter.connector.PropertiesUtf8Loader
import com.taskadapter.connector.definition.WebConnectorSetup

object TrelloTestConfig {
  private val TEST_PROPERTIES = "trello.properties"
  private val properties = PropertiesUtf8Loader.load(TEST_PROPERTIES)

  def getConfig = {
    val config = new TrelloConfig();
    config.setBoardId(properties.getProperty("boardId"))
    config.setBoardName("")
    config
  }

  def getSetup = WebConnectorSetup.apply(TrelloConnector.ID, "label1",
    "", "", properties.getProperty("apikey"),
    true, properties.getProperty("token"))
}

File: connectors/connector-trello/src/test/java/com/taskadapter/connector/trello/TrelloTestConfig.java
package com.taskadapter.connector.trello;

import com.taskadapter.connector.PropertiesUtf8Loader;
import com.taskadapter.connector.definition.WebConnectorSetup;

import java.util.Properties;

public class TrelloTestConfig {

    private static final String TEST_PROPERTIES = "trello.properties";
    private static final Properties properties = PropertiesUtf8Loader.load(TEST_PROPERTIES);

    static TrelloConfig getConfig() {
        var config = new TrelloConfig();
        config.setBoardId(properties.getProperty("boardId"));
        config.setBoardName("");
        return config;
    }

    static WebConnectorSetup getSetup() {
        return WebConnectorSetup.apply(TrelloConnector.ID, "label1",
                "", "", properties.getProperty("apikey"),
                true, properties.getProperty("token"));
    }
}
